### PR TITLE
Log Streaming completed | Say hello to ReactJS

### PR DESCRIPTION
**Pull request by morpheyesh**

### CODE REVIEW

**Comment by indykish** [comment]
@morpheyesh Cool dude. We now have a hang of ReactJS.  There are conflicts, fetch merge the code.
Glad that we are moving out of  `screwy angularjs`

Educate @arunkumarsekar06  @vijaykanthm28 @rajthilakmca @LogeshEswar

File: app/assets/javascripts/components/socketService.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+console.log("inside socketservice");
+	var service = {};
+
+
+	function init() {
+		service = {};
+
+	}
+
+console.log("init-->");
```
Comment (indykish) [inline app/assets/javascripts/components/socketService.js]: @morpheyesh Probably remove the console.log stufff.

File: app/views/catalogs/_logs.html.erb [hunk 2]
```diff
@@ -75,3 +33,22 @@
     		$.SocketURL = "<%= Ind.websocket_tap %>";
      })($);
 	</script>
+
+<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
```
Comment (indykish) [inline app/views/catalogs/_logs.html.erb]: Is this the latest socket.io version  ? or move it up to the latest ? 
Can we not just use a `gem` as its called from the `html.erb` ?

File: app/views/catalogs/_logs.html.erb [hunk 3]
```diff
@@ -75,3 +33,22 @@
...
+  var socket = io.connect($.SocketURL);
+  console.log(socket);
+
+    $('.socket_disconnect').click(function(){
+         socket.disconnect();
+    });
+
+
+		socket.on("disconnect", function(){
+        console.log("client disconnected from server");
```
Comment (indykish) [inline app/views/catalogs/_logs.html.erb]: Again console.log, please remove them.

File: config/nilavu.yml [hunk 4]
```diff
@@ -37,7 +43,7 @@ defaults: &defaults
 ##
 ## Acceptable values:
 ##   - text, e.g. localhost, api.megam.io
-  websocket_tap: console.megam.io:7000
+  websocket_tap: localhost:7000
```
Comment (indykish) [inline config/nilavu.yml]: we will to have to rename this propery, as there will no `tap`

File: app/views/catalogs/_logs.html.erb (hunk 3 above)
Comment (morpheyesh) [inline app/views/catalogs/_logs.html.erb]: Woops! Will refine and push it tomorrow.  :)

File: app/assets/javascripts/components/socketService.js (hunk 1 above)
Comment (morpheyesh) [inline app/assets/javascripts/components/socketService.js]: This file is not required. will remove it.

File: app/views/catalogs/_logs.html.erb (hunk 2 above)
Comment (morpheyesh) [inline app/views/catalogs/_logs.html.erb]: Gem does not seem to work, will try again. Moving to 1.3.7.

File: config/nilavu.yml (hunk 4 above)
Comment (morpheyesh) [inline config/nilavu.yml]: renamed it to http_logs

**Comment by indykish** [comment]
@morpheyesh  Its much cleaner. Lets review it on monday and merge it.

**Pull request merged**